benefits/settings.py

```diff
@@ -168,14 +168,16 @@ def _filter_empty(ls):
 
 # OAuth configuration
 
-OAUTH_CLIENT_NAME = os.environ.get("DJANGO_OAUTH_CLIENT_NAME")
+OAUTH_AUTHORITY = os.environ.get("DJANGO_OAUTH_AUTHORITY", "http://example.com")
+OAUTH_CLIENT_NAME = os.environ.get("DJANGO_OAUTH_CLIENT_NAME", "benefits-oauth-client-name")
+OAUTH_CLIENT_ID = os.environ.get("DJANGO_OAUTH_CLIENT_ID", "benefits-oauth-client-id")
```

Comment:
At risk of over-complicating things: should it to throw an error if settings like these aren't configured in production? Not a blocker; happy to create a follow-up issue.

Reply:
We are currently doing that check, but it is deep within the flow (it's when we [show the button to the user](https://github.com/cal-itp/benefits/blob/dev/benefits/eligibility/views.py#L95-L96)). I think it makes sense to check it more upfront, though we'd need to figure out the appropriate place to do it. A follow-up issue sounds good to me.

Reply:
> A follow-up issue sounds good to me.

Er, that is - if the consensus is that we want to do this 😄